string serial_port
---
bool status
